src/hooks/useReportScrollManager/types.ts

```diff
@@ -0,0 +1,10 @@
+import {ActionListContextType} from '../../pages/home/ReportScreenContext';
+
+type ReportScrollManagerData = {
+    ref: ActionListContextType;
+    scrollToIndex: (index: number, isEditing: boolean) => void;
+    scrollToBottom: () => void;
+};
+
+// eslint-disable-next-line import/prefer-default-export
+export type {ReportScrollManagerData};
```

Comment:
You can export it as default, we do it already in other types.ts files.
```suggestion
export default ReportScrollManagerData;
```

Reply:
Done